### ISSUE TITLE
Problems in the ConcatFusionFactor of common.py

### ISSUE THREAD

**Issue by CherrPeac** [WindVChen/DRENet#15]
Hello, while reading your paper, you wrote: "Suppose that there are two layers C1 and C2 in the bottom-up pathway from shallow to deep, and two layers P1 and P2 in the top-down pathway corresponded. In the original FPN, P1 = C1 + upsample(P2); after we add the scale layer, it turns to be P1 = C1 + α× upsample(P2), where α is a learnable parameter." in methodology. I guess this is the function of the ConcatFusionFactor module, but in this module, this factor is 1.0 and It doesn't seem to change, or does it default to 1.0 and change elsewhere?

```
class ConcatFusionFactor(nn.Module):
    # Concatenate a list of tensors along dimension
    def __init__(self, dimension=1):
        super(ConcatFusionFactor, self).__init__()
        self.d = dimension
        self.factor=torch.nn.Parameter(torch.FloatTensor([1]))

    def forward(self, x):
        x[0] = x[0]*self.factor
        return torch.cat(x, self.d)
```

**Comment by CherrPeac** [comment]
I am so stupid. It can be updated through backpropagation.

**Comment by WindVChen** [comment]
Hi @CherrPeac ,

Yes. The factor can be updated through backpropagation. It may also need to be noted that despite some benefit on the final performance, sometimes updating it will make the training phase unstable. Thus we currently freeze it by default (i.e., fix the factor to 1.0) in the [code](https://github.com/WindVChen/DRENet/blob/c4cec875e538fe85bc9d882ca9e3da74f18167b6/train.py#LL113C10-L113C10).
